agents/bug_reporter/main.py

```diff
@@ -0,0 +1,97 @@
+"""This is a Fixie Agent that files issues in GitHub."""
+
+import os
+
+import fixieai
+from github import Github
+
+# Add a GITHUB_ACCESS_TOKEN to the .env file in this directory before deploying the agent.
+# The token needs read/write access to the repo you want to file issues against.
+GITHUB_ACCESS_TOKEN = os.environ.get("GITHUB_ACCESS_TOKEN")
+if not GITHUB_ACCESS_TOKEN:
+    raise ValueError("GITHUB_ACCESS_TOKEN is not set!")
+
+gh = Github(GITHUB_ACCESS_TOKEN)
+# This example Agent files issues against the Fixie SDK repo. Change the following to use a different
+# repo.
+REPO = gh.get_repo("fixie-ai/fixie-sdk")
+LABEL = REPO.get_label("user-reported")
+
+BASE_PROMPT = """I am an agent that files bug reports to GitHub."""
+
+FEW_SHOTS = """
```

Comment:
Should we have a fewshot where most of the details are included in the initial Q and thus requires less back and forth?

Reply:
That is the third one, even if details are provided up front I want to make sure the user gets a chance to add details.